azext_iot/central/commands_device.py

```diff
@@ -382,42 +347,20 @@ def add_children(
     children_ids: List[str],
     token=None,
     central_dns_suffix=CENTRAL_ENDPOINT,
-    api_version=ApiVersion.v1.value,
+    api_version=API_VERSION,
 ):
     from uuid import uuid4
 
-    if api_version != ApiVersion.v1_1_preview.value:
-        raise InvalidArgumentValueError(
-            (
-                "Adding children devices to IoT Edge is still in preview "
-                "and only available for Api version >= 1.1-preview. "
-                'Please pass the right "api_version" to the command.'
-            )
-        )
-
     provider = CentralDeviceProvider(
         cmd=cmd, app_id=app_id, token=token, api_version=api_version
     )
-    rel_name = get_downstream_rel_name(
-        cmd,
-        app_id=app_id,
-        device_id=device_id,
-        token=token,
-        central_dns_suffix=central_dns_suffix,
-        api_version=api_version,
-    )
-
-    if not rel_name:
-        raise ResourceNotFoundError(
-            f'Relationship name cannot be found in the template for device with id "{device_id}"'
-        )
 
     return [
         provider.add_relationship(
             device_id=device_id,
             target_id=child_id,
             rel_id=str(uuid4()),
-            rel_name=rel_name,
+            rel_name=None,
```

Comment:
do you still need this or can you remove it since you are setting this to None

Reply:
No we don't. Actually the rel_name retired long time ago and somehow we didn't remove it. 😓 I'll remove this.

Reply:
Removed.